test/unit/nedap/speced/def/spec_assertion.cljc

```diff
@@ -53,3 +54,23 @@
   (is (spec-assertion-thrown? 'number? (returns-number-fn "1234")))
   (is (spec-assertion-thrown? 'string? (accepts-string (->ExampleRecord) 1234)))
   (is (spec-assertion-thrown? 'string? (returns-string (->ExampleRecord) 1234))))
+
+(defn parameterized-defn [spec x]
+  (check! spec x)
+  x)
+
+#?(:clj (defmacro when-not-u-s-1-0-0
+          {:style/indent 0}
+          [& body]
+          (when-not (-> "nedap.speced.def.testing.utils-spec-dep" System/getProperty #{"1.0.0"})
+            `(do
```

Comment:
`do` can be removed

Reply:
Tried, and then remembered why it's there. Note that the when-not happens at macroexpasion time: at no point a `(when-not x (do)` form is emitted.